experiments/standalone/Snow/snowmip_simulation.jl

```diff
@@ -57,6 +69,8 @@ Y, p, coords = ClimaLand.initialize(model)
 
 # Set initial conditions
 Y.snow.S .= FT(SWE[1]) # first data point
+#Y.snow.Z .= FT(depths[1]) #uncomment if using dens_neural, not dens_constant
+p.snow.ρ_snow .= FT(SWE[1] / depths[1] * 1000.0)
```

Comment:
you dont need to set this, it is set in `set_initial_cache!`

Reply:
Note to myself to check a model simulation without this line, I think I ran into problems without it